src/plugin/plugin_pdo.rs

```diff
@@ -198,8 +198,17 @@ fn after_hook(
             );
         }
     } else if let Some(obj) = return_value.as_mut_z_obj() {
-        if obj.get_class().get_name() == &"PDOStatement" {
-            return after_hook_when_pdo_statement(get_this_mut(execute_data)?, obj);
+        let cls=obj.get_class().get_name().to_str()?;
+        match cls {
+            "PDOStatement" => {
```

Comment:
Use `"PDOStatement" | r"Doctrine\DBAL\Driver\PDO\Statement"` here so you can merge the condition?